src/operator/mshadow_op.h

```diff
@@ -45,14 +45,10 @@ namespace mshadow_op {
 __constant__ const float PI = 3.14159265358979323846;
 __constant__ const float SELU_ALPHA = 1.6732632423543772848170429916717;
 __constant__ const float SELU_LAMBDA = 1.0507009873554804934193349852946;
-__constant__ const float GELU_CUBIC_CONSTANT = 0.044715;
```

Comment:
Can we define `__constant__ const float SQRT_2 = 1.4142135623730950488;` and avoid computing `math:sqrt(2)`?

Reply:
Done